votes/TEMPLATE.md

```diff
@@ -4,6 +4,10 @@
 
 Replace this content with the proposal.
 
+## Deadline
+
+The vote will be open for at least {duration} unless there is an objection or not enough votes.
```

Comment:
`duration` is not decorated correctly.

Reply:
Do you mean `${duration}`? I'm ok for both.

Reply:
I mean when I clicks `Preview` I can't see specific `duration`.

Reply:
That should be populated for the author who creates a concrete vote. It may be 3 days or 6 days, for example. There is no automatic substitution for the placeholder. It is rendered normally, BTW.

![image](https://user-images.githubusercontent.com/18818196/132205222-0df5d4ac-b7ec-4fca-aaf3-816ca1e05a94.png)

Reply:
Can you add comment to the sentence?

Reply:
Sounds reasonable. Let me think of it.

Reply:
Updated as a22075addff6e03837c48a09d4f92d1c19bd4ff6. PTAL